pkg/parcacol/normalizer.go

```diff
@@ -167,7 +169,7 @@ func LabelNamesFromSamples(
 }
 
 // TODO: support num label units.
-func LabelsFromSample(takenLabelNames map[string]string, stringTable []string, plabels []*pprofpb.Label) (map[string]string, map[string]int64) {
+func LabelsFromSample(takenLabels map[string]string, stringTable []string, plabels []*pprofpb.Label) (map[string]string, map[string]int64) {
 	labels := map[string][]string{}
 	labelNames := []string{}
```

Comment:
@brancz we collect `labelNames` and go over them in sorted order later on. We return map `resLabels` so I wonder whether we need to do that. We don't do it in `LabelNamesFromSamples`. All tests (+ manual test) pass if I remove `labelNames` and just iterate over the `labels` keys.

Reply:
it's so we have a consistent order if we see the same labelset again, I agree we should have a test for this case, and we need to fix it in the other place, neither of that needs to happen in this PR though

Reply:
got it, thank you!